calculation/gmhazard_calc/gmhazard_calc/directivity/validation/plots.py

```diff
@@ -158,19 +159,23 @@ def plot_fdi(
         Each point of the srf fault in an array with the format [[lon, lat, depth],...]
     output_ffp: Path
         Ffp to the location of the output plot image
+    title: string, optional
+        The title for the plot
     """
     fig, (ax1) = plt.subplots(1, 1, figsize=(21, 13.5), dpi=144)
 
-    m = ax1.contourf(x, y, np.exp(fdi), cmap="bwr", levels=11)
-    ax1.contour(x, y, np.exp(fdi), colors="k", linewidths=0.3, levels=11)
+    c = ax1.pcolormesh(x, y, np.exp(fdi), cmap="bwr")
     ax1.scatter(
         lon_lat_depth[:, 0][::2],
         lon_lat_depth[:, 1][::2],
         c=lon_lat_depth[:, 2][::2],
         label="srf points",
         s=1.0,
     )
-    plt.colorbar(m, pad=0.01)
-    ax1.set_title(r"$f_Di$")
+    cb = plt.colorbar(c)
+    cb.set_label("fD")
+    plt.ylabel("Latitude")
```

Comment:
You have used the `ax` approach for all other matplotlib calls, so I suggest to do the same in this case, i.e. `ax1.set_ylabel(...)`